node/node.go

```diff
@@ -1576,12 +1575,16 @@ func (app *App) setupDBs(ctx context.Context, lg log.Log) error {
 	if err != nil {
 		return fmt.Errorf("failed to load migrations: %w", err)
 	}
-	sqlDB, err := sql.Open("file:"+filepath.Join(dbPath, dbFile),
+	dbopts := []sql.Opt{
 		sql.WithMigrations(migrations),
 		sql.WithConnections(app.Config.DatabaseConnections),
 		sql.WithLatencyMetering(app.Config.DatabaseLatencyMetering),
-		sql.WithV5Migration(util.ExtractActiveSet),
-	)
```

Comment:
i think it is safe to drop that. that version won't be able to participate in the protocol, due to many changes that we introduced. for example those limits

Reply:
Yes, but I mean it would still make sense to add a note to the changelog / release that people can only upgrade to 1.3.0 if they were running 1.2.x before. Just in case someone tries to rejoin the network after being offline for some time 🙂